src/TTextEdit.cpp

```diff
@@ -1019,7 +1019,7 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
             mudletEvent.mArgumentList.append(QString::number(3));
             break;
         default: // TODO: What about those of us with more than three mouse buttons?
-            mudletEvent.mArgumentList.append(0);
+            mudletEvent.mArgumentList.append(nullptr);
```

Comment:
This appends the empty string, so it should be `QString()`, if you don't want the behavior to change.